stumpy/gpu_aamp.py

```diff
@@ -263,7 +304,7 @@ def _gpu_aamp(
     See Table II, Figure 5, and Figure 6
     """
     threads_per_block = config.STUMPY_THREADS_PER_BLOCK
-    blocks_per_grid = math.ceil(k / threads_per_block)
+    blocks_per_grid = math.ceil(w / threads_per_block)
```

Comment:
This was the issue! It is just a silly mistake. I forgot to update this line. Since we now use `k` for `top-k`, I had to change the name of variable but I forgot to do so for this line.

Reply:
Excellent. I'm glad that it makes logical sense and things are still consistent between normalized and non-norm